trino/README.md

```diff
@@ -23,7 +23,22 @@ For Agent v7.33.0+, follow the instructions below to install the Trino check on
 
 ### Configuration
 
-1. Edit the `trino.d/conf.yaml` file, in the `conf.d/` folder at the root of your
+1. This integration uses JMX to collect metrics from Trino. For the integration to be able to do that you need to enable JMX on Trino by adding the following properties to Trino's `config.properties` file:
+
+   ```properties
+   jmx.rmiregistry.port=9080
+   jmx.rmiserver.port=9081
+   ```
+
+   Also add the following properties to Trino's `jvm.config` file:
+
+   ```
+   -Dcom.sun.management.jmxremote
+   -Dcom.sun.management.jmxremote.port=9080
+   -Dcom.sun.management.jmxremote.rmi.port=9081
+   ```
```

Comment:
Can we also clarify in case of Helmchart do we need to add these properties in both config-map coordinator and worker or only in any on of them?